### ISSUE TITLE
Inquiry for training details about diffusion policy

### ISSUE THREAD

**Issue by lucywang720** [YanjieZe/3D-Diffusion-Policy#115]
Hi,

Thanks for your great work! I am trying to reproduce the results of diffusion policy in your paper, but it seems a little strange. So i am here to ask for some details:
1. Is there any difference between the hyperparameters of dp3 and diffusion policy, like lr, ...
2. Is the approach of calculating success rate different from dp3? e.g. 'We run 3 seeds for each experiment with seed number 0, 1, 2. For each seed, we evaluate 20 episodes every 200 training epochs and then compute the average of the highest 5 success rates. We report the mean and std of success rates across 3 seeds.'
3. Is the architecture of dp3 and diffusion policy different, except for the type of input and the dimension of each layer? 

Thank you very much!

**Comment by YanjieZe** [comment]
Hi, thank you for your interest in our work.  all the same. except that we use sample prediction instead of noise prediction for DDIM.
